spec/index.bs

```diff
@@ -2218,6 +2211,9 @@ interface GPUTextureUsage {
 
             1. Issue the following steps on the [=Device timeline=] of |this|:
                 <div class=device-timeline>
+                    1. If |descriptor|.{{GPUTextureDescriptor/format}} is a {{GPUTextureFormat}} that requires a feature,
```

Comment:
nit: link to `[[#texture-format-caps]]` so there's a source to figure out whether a format requires a feature.

Reply:
Ah, I saw your other comment. IMO better to link to both. The comment in the IDL for GPUTextureFormat is informative and not really written in a very detailed/normative way.

Reply:
Done.